pkg/api/handlers/libpod/containers.go

```diff
@@ -178,7 +178,6 @@ func KillContainer(w http.ResponseWriter, r *http.Request) {
 func WaitContainer(w http.ResponseWriter, r *http.Request) {
 	exitCode, err := utils.WaitContainer(w, r)
 	if err != nil {
-		utils.InternalServerError(w, err)
```

Comment:
Shouldn't err be reported somewhere?

Reply:
the error is sent down the pipe in utils.WaitContainer ...